# 3 buttons on the right side (Windows-like layout)
--- Adwaita/resource/layout/uinavigatorpanel.layout
+++ Adwaita/resource/layout/uinavigatorpanel.layout
@@ -2,9 +2,9 @@
 {
 	layout
 	{
		region { name="left" x=0 y=7 width=166 height=34 }
-		region { name="tabs" margin-left=170 margin-right=170 }
+		region { name="tabs" margin-left=230 margin-right=244 }
 
		place { control="label_console" region="left" x=7 y=0 width=34 height=42 }
		place { control="back,forward" region="left" start="label_console" x=7 y=0 width=34 height=34 spacing=6 }
 

--- Adwaita/resource/steamscheme.res
+++ Adwaita/resource/steamscheme.res
@@ -15,7 +15,6 @@
 
 			frame_maximize
 			{
-				visible	0
 				xpos	-999
				ypos	0
				wide	0
@@ -25,7 +24,6 @@
 
 			frame_minimize
 			{
-				visible	0
 				xpos	-999
				ypos	0
				wide	0
